scripts/apim.test.ts

```diff
@@ -0,0 +1,72 @@
+import { fail } from "assert";
+
+import axios, { AxiosResponse } from "axios";
+
+
+const APP_NAME =
+  process.env.APP_NAME
+
+const TEST_URL =
+  process.env.TEST_URL
+
+describe("APIM Test", () => {
```

Comment:
I don't think these are the right tests, I would just clear it mostly out and have a dummy test case for now?